SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1172,11 +1163,33 @@ void USpatialActorChannel::UpdateSpatialPosition()
 			return;
 		}
 	}
+	
+	// Check that the Actor has satisfied both minimum thresholds OR either of the maximum thresholds
```

Comment:
Unsure about the world minimum but I'm not sure how much it matters. They aren't actually a minimum in the same way the maximum is a maximum. They could be the "lower" thresholds but I'm not sure that actually makes it clearer.

Reply:
I think I see what your point is, but not sure how to improve the names to make it more clear?

Reply:
You could could call it the "lower" threshold. I'm not 100% sure of that being better but it gets the idea across. On which note it might be worth putting somthing in settings that enforces that they are actually lower. I'm not sure we get much out of checking it but still.

Reply:
Not sure if this is a valid use case, but say that maybe a user wants to update an Actor based on either only the distance, or only the elapsed time, so he would achieve that by for example setting MaxTime = 10, MaxDist = 10, MinTIme=100, MinDist= 100. Is this something we would like to prevent users from doing?

Also this 'use case' kind of makes it more necessary for a change in the variable names?

Reply:
I think that would be a mistake. There is no way the min stuff ever gets used as for both of those thresholds to be met, one of the other two must first. So that would just be there to effectively say you want to ignore the "min" values.

The main reason I don't like Min is because they aren't minimums though.

Reply:
I see, I will try to come up with something that is more informative and does not contain Min. Anyways, naming turns out to be unexpectedly hard 😄!